core/src/main/java/org/elasticsearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -135,7 +135,7 @@ private static BulkItemResultHolder executeIndexRequest(final IndexRequest index
         if (indexResult.hasFailure()) {
             return new BulkItemResultHolder(null, indexResult, bulkItemRequest);
         } else {
-            IndexResponse response = new IndexResponse(primary.shardId(), indexRequest.type(), indexRequest.id(),
+            IndexResponse response = new IndexResponse(primary.shardId(), indexRequest.type(), indexRequest.id(), indexRequest.source().utf8ToString(),
```

Comment:
I think the issue wants this to the be index we created the document in, not the request.